gatsby-theme-landing-page/src/components/head.js

```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 import { Helmet } from "react-helmet";
 
-export default function Head({ title, description }) {
+export default function Head({ title, description, image }) {
+  const imageUrl = image ? `https:${image.file.url}` : "";
```

Comment:
Guessing this returns a schemaless URL; does that not work with Twitter/etc.?

Reply:
Weirdly contentful returns it with the leading double slash, e.g. `//images.contentful.com/etc`.

I'm following the pattern we use for our .com SEO component